### DISCUSSION TITLE
Download Manifest File

### DISCUSSION THREAD

**Discussion by alec-hs** [ValvePython/steamctl#63] [Q&A] [answered]
Is it possible to use this to download a manifest file? Looking to replace depot downloader with this as it is cross platform, but I need to be able to download the actual manifest files not just the game files.

I've tried a few combinations of the depot command but can't seem to get what I need. 

Thanks

**Comment by SlejmUr** [comment, marked as answer]
Yep it download manifest file in the cache.
AppData/Roaming or Local. It download and unpack it so it will be not 100% same as you download from steam.

**Comment by alec-hs** [comment]
Not the format I was looking for but It led me down a path to just build a JSON version of the file using the steam python package instead.
